.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "workbench.colorTheme": "Solarized Dark",
+    "workbench.colorCustomizations": {
+        "activityBar.foreground": "#ff0000",
+        "activityBarBadge.foreground": "#000000",
+        "activityBarBadge.background": "#ff000095",
+        "sideBar.background": "#200000",
+    },
+    "python.pythonPath": "C:\\Users\\sean.micek\\AppData\\Local\\Continuum\\anaconda3\\envs\\ccheck48\\python.exe"
+}
```

Comment:
Could you rebase to remove this file?

Reply:
EDIT, was looking at Dalton's repo, not the IOOS one. Disregard and/or request any further fixes over on the IOOS repository.